aten/src/ATen/core/LegacyTypeDispatch.h

```diff
@@ -139,11 +139,6 @@ class CAFFE2_API LegacyTypeDispatch {
 
 CAFFE2_API LegacyTypeDispatch& globalLegacyTypeDispatch();
```

Comment:
It won't let me put a comment there, but can you remove the `at::NonVariableTypeMode::is_enabled()` call from `initializeLegacyTypeDispatchFor` and `legacyTensorType`? Also expunge other call sites that are called in conjunction with `is_variable()`